js/main.js

```diff
@@ -0,0 +1,187 @@
+// Navbar
+const menuOpen = document.getElementById('menuOpen');
+const menuClose = document.getElementById('menuClose');
+
+const showNavbar = () => {
+  const navbar = document.querySelector('.navbar__container');
+
+  if (window.matchMedia('(max-width: 768px)').matches) {
+    menuOpen.style.display = 'none';
+    menuClose.style.display = 'block';
+    document.body.style.overflow = 'hidden';
+    navbar.style.left = '0';
+  }
+};
+menuOpen.addEventListener('click', showNavbar);
+
+const hiddeNavbar = () => {
+  const navbar = document.querySelector('.navbar__container');
+
+  if (window.matchMedia('(max-width: 768px)').matches) {
+    menuClose.style.display = 'none';
+    menuOpen.style.display = 'block';
+    document.body.style.overflow = 'scroll';
+    navbar.style.left = '-100%';
+  }
+};
+menuClose.addEventListener('click', hiddeNavbar);
+
+// Main Program
+const programs = [
+  {
+    image: './images/program_icon_01.png',
+    title: 'Lecture',
+    description: 'Listen to the speakers from variuos countries abort the   messages of sharing.',
+  },
+  {
+    image: './images/program_icon_02.png',
+    title: 'CC Exhibition',
+    description: 'Appreciate variuos creations applying CC license of artists, from Art Center Nabi.',
+  },
+  {
+    image: './images/program_icon_03.png',
+    title: 'Forum',
+    description: 'Have te time to share your thoughts and opinions with experts for each topic.',
+  },
+  {
+    image: './images/program_icon_04.png',
+    title: 'Workshop',
+    description: 'Try creating your own work using open source license rather than just watching.',
+  },
+  {
+    image: './images/program_icon_05.png',
+    title: 'CC Ignite',
+    description: 'Get opportunities to network with CC affiliates around the world, aslo after the summit.',
+  },
+];
+
+const printProgram = () => {
+  const programSection = document.getElementById('programSection');
+  programs.forEach((program) => {
+    const ul = document.createElement('ul');
+    ul.classList.add('program__list__container');
+    ul.innerHTML = `
+        <li class="program__list__item">
+            <figure class="program__image__container">
+                <img src="${program.image}" alt="lecture_img"
+                class="program__list__image">
+            </figure>
+            <h3 class="program__list__title">${program.title}</h3>
+            <p class="program__list__paragraph">
+                ${program.description}
+            </p>
+        </li>
+        `;
+    programSection.appendChild(ul);
+  });
+};
+printProgram();
```

Comment:
As mentioned above the code for printing the programs and the featured speakers follows a similar pattern. Kindly I suggest you create a generic function that takes the data and the target container as parameters. This function could handle the creation of the elements and appending them to the container, reducing code duplication. 😄